src/components/ProductCardRequest/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, useMemo } from 'react';
+import ProductCard from 'components/ProductCard';
+import {
+  useGetProductByIdQuery,
+  useGetProductImagesQuery,
+} from 'redux/productsApi';
+
+interface ProductCardRequestProps {
+  id: string;
+}
+
+const ProductCardRequest: FC<ProductCardRequestProps> = ({ id }) => {
+  const { data } = useGetProductByIdQuery({ id });
+  const images = useGetProductImagesQuery(data?.files);
```

Comment:
it seems you can move images inside the useMemo

Reply:
Hooks cannot be called inside useMemo(...)